### PR TITLE
Feature gate "SubnetsClusterTagCheck" to disable subnet cluster tag check

### PR DESCRIPTION
**Pull request by franklinpashok**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -104,3 +104,103 @@ to a Kubernetes Cluster.
 | <a name="output_iam_role_name"></a> [iam\_role\_name](#output\_iam\_role\_name) | Name of IAM role |
 | <a name="output_iam_role_path"></a> [iam\_role\_path](#output\_iam\_role\_path) | Path of IAM role |
 | <a name="output_iam_role_unique_id"></a> [iam\_role\_unique\_id](#output\_iam\_role\_unique\_id) | Unique ID of IAM role |
+
```
Comment (lawliet89) [inline README.md]: Can you delete the ones above? They are repeated.

**Review by lawliet89 — CHANGES_REQUESTED** [review]
Bump chart to 1.4.4 and image version to 2.4.3 pls.

File: README.md (hunk 1 above)
Comment (lawliet89) [reply README.md]: The README still has duplicate tf-docs generated stuff @franklinpashok

Comment (franklinpashok) [reply README.md]: remove the duplicates in readme. yes the tf-docs if run after deleting it updated again. Now fixed

**Review by lawliet89 — APPROVED** [review]

**Pull request merged**